rsts/community/troubleshoot.rst

```diff
@@ -1,126 +1,128 @@
 .. _troubleshoot:
 
+=====================
 Troubleshooting Guide
----------------------
+=====================
 
 .. tags:: Troubleshoot, Basic
 
-.. admonition:: Why did we craft this guide?
+The content in this section will help Flyte users isolate the most possible causes for some of the common issues that could arise while getting started with the project.
 
-    To help streamline your onboarding experience as much as possible, and sort out common issues.
+Before getting started, collect the following information from the underlying infrastructure:
 
-Here are a couple of techniques we believe could help get you up and running in no time! 
+- Capture the ``Status`` column from the output of: 
 
-Troubles With ``flytectl sandbox start``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-- The process hangs at ``Waiting for Flyte to become ready...`` for a while; OR 
-- It ends with a message ``Timed out while waiting for the datacatalog rollout to be created``.
+.. prompt:: bash $
 
-How Do I Debug?
-"""""""""""""""
+ $ kubectl describe pod <PodName> -n <namespace>
```

Comment:
might be worth calling out that `<PodName>` will typically correspond to the node execution string that you can find in the UI